src/libs/Network.js

```diff
@@ -34,7 +34,8 @@ function processNetworkRequestQueue() {
 
         // Make a simple request every second to see if the API is online again
         HttpUtils.xhr('Get', {doNotRetry: true})
-            .then(() => NetworkConnection.setOfflineStatus(false));
+            .then(() => NetworkConnection.setOfflineStatus(false))
+            .catch(() => {});
```

Comment:
Another option would be to add a log here instead of doing nothing like we did for the timezone API call below. Then we will at least know which error has been thrown.

Reply:
So this piece of code is to just ping the server and set the state accordingly. I think we can catch this as least.

Reply:
ok. I will log the error message.